README.rst

```diff
@@ -97,6 +98,34 @@ advanced settings.
    ``"limesurvey"`` to the policy value list.
 3. Click the "Save changes" button.
 
+
+Instructor Dashboard
+********************
+In the instructor dashboard, you can see a table containing a list
+of all the blocks added to the course by the instructor.
+This table contains the block's name and a button to access the
+administrator according to the URL of the service defined in the block.
+
+
+Behavior
+**************
+1. When the URLs are not configured, the service will display an error message.
+2. When the survey ID is not valid, the service will display an error message.
+
+
+Tips
```

Comment:
I think this would be clearer if we add a section called 'Configuring LimeSurvey', where we mention anonymous surveys, close/open access, attribute_1

Reply:
And we can add a section called: Configuring the LimeSurvey component (in the CMS) or a title like it. Where we attach screenshots of the view in studio, the fields they can configure, etc.

We can also have a section called:
View from the LMS
-------------------------

As student
--------------

As instructor
----------------
Here we add what we have on the instructor